src/Cron/HoursField.php

```diff
@@ -167,37 +167,37 @@ public function increment(DateTimeInterface &$date, $invert = false, $parts = nu
         if (! $invert) {
             if ($originalHour >= $target) {
                 $distance = 24 - $originalHour;
-                $date = $this->timezoneSafeModify($date, "+{$distance} hours");
+                $date = $this->timezoneSafeModify($date, "{$distance} hours");
 
                 $actualDay = (int)$date->format('d');
                 $actualHour = (int)$date->format('H');
                 if (($actualDay !== ($originalDay + 1)) && ($actualHour !== 0)) {
                     $offsetChange = ($previousOffset - $date->getOffset());
-                    $date = $this->timezoneSafeModify($date, "+{$offsetChange} seconds");
+                    $date = $this->timezoneSafeModify($date, "{$offsetChange} seconds");
                 }
 
                 $originalHour = (int)$date->format('H');
             }
 
             $distance = $target - $originalHour;
-            $date = $this->timezoneSafeModify($date, "+{$distance} hours");
+            $date = $this->timezoneSafeModify($date, "{$distance} hours");
         } else {
             if ($originalHour <= $target) {
                 $distance = ($originalHour + 1);
-                $date = $this->timezoneSafeModify($date, "-" . $distance . " hours");
+                $date = $this->timezoneSafeModify($date, (-$distance) . " hours");
 
                 $actualDay = (int)$date->format('d');
                 $actualHour = (int)$date->format('H');
                 if (($actualDay !== ($originalDay - 1)) && ($actualHour !== 23)) {
                     $offsetChange = ($previousOffset - $date->getOffset());
-                    $date = $this->timezoneSafeModify($date, "+{$offsetChange} seconds");
+                    $date = $this->timezoneSafeModify($date, "{$offsetChange} seconds");
                 }
 
                 $originalHour = (int)$date->format('H');
             }
 
             $distance = $originalHour - $target;
-            $date = $this->timezoneSafeModify($date, "-{$distance} hours");
+            $date = $this->timezoneSafeModify($date, (-$distance) . " hours");
```

Comment:
Accordingly to [relative formats docs](https://www.php.net/manual/en/datetime.formats.php#datetime.formats.relative) this should be "hours ago"

What you wrote is just different syntax, it is being interpreted the same way - see https://3v4l.org/KRVLH

Reply:
@mabar Yes, right, but it will work incorrect if $distance is negative.
For example if we are calling `modify("--10 hours)`, we expect it like "+10 hours", but actually since php8.2 it will be working like "-10 hours": https://3v4l.org/X1XJd#v8.3.4